ark/utils/data_utils_test.py

```diff
@@ -512,7 +512,14 @@ def test_split_img_stack():
         junk_img_chan_last = np.zeros((1024, 1024, 10))
         io.imsave(os.path.join(stack_dir, "channel_data.tif"), junk_img_chan_last)
 
-        data_utils.split_img_stack(stack_dir, output_dir, stack_list, indices, names, channels_first=False)
+        data_utils.split_img_stack(
+            stack_dir,
```

Comment:
I think for stuff that's this short, indenting every line just makes the code take up more space and isn't that helpful